docs/best-practices/continuous-integration.md

```diff
@@ -446,3 +446,60 @@ The extension can be configured to install the latest version of _fastlane_ or a
+1. Finalize the setup by checking the appropriate [build options](https://developer.nevercode.io/docs/build-configuration#section-general-build-settings) and click **Save and start build**.
+
+## External dependencies
+
+Nevercode does not automatically install CocoaPods, Carthage or any other external dependencies for _fastlane_ projects. Please use the _fastlane_'s built-in [actions](https://docs.fastlane.tools/actions/), such as [`cocoapods`](https://docs.fastlane.tools/actions/cocoapods/) and [`carthage`](https://docs.fastlane.tools/actions/carthage/), to gain control of that.
+
+## Code signing
+
+To perform code signing on your iOS app, you can either follow the best practices laid out in [`fastlane` docs](https://docs.fastlane.tools/codesigning/getting-started/#using-match), or you could also make use of [Nevercode's standard code signing methods](https://developer.nevercode.io/docs/code-signing).
+
+When invoking [`match`](https://docs.fastlane.tools/actions/match/) from the `Fastfile`, keep in mind that you need to **grant access to the credentials repository** and expose the **_match_ passphrase** during the build. Both can be securely achieved thanks to the well designed environment variable support of _fastlane_ and Nevercode's support for several SSH keys for authentication.
+
+In your app settings on Nevercode, navigate to the **Environment** tab to manage environment variables and files:
+
+* _match_ passphrase can be defined as a `MATCH_PASSWORD` environment file — this will be automatically detected by _fastlane_ and used to decrypt the credentials repository.
+* You can upload the SSH key for cloning the credentials repository as an environment file, i.e. `MATCH_SSH_KEY`. All environment files whose variable name has the suffix `_SSH_KEY` will be automatically added to the SSH agent and will be ready for use during the whole build process.
+
+## Run tests
+
+Testing in Nevercode with _fastlane_ is 100% automatic. By default, [`scan`](https://docs.fastlane.tools/actions/scan/) is used to invoke the test run, but you can easily swap it out for any other `lane` that you have configured for your test runs. Test results are automatically collected and you don't need to do anything to convert the results from one format to another or place them in a special location on the build machine.
+
+### Warning
+In case you wish to run tests as part of the main build step, we suggest that you use [Trainer](https://github.com/KrauseFx/trainer#use-with-fastlane) to generate reliable JUnit results from the test run and put the results in a directory where we can discover them:
+
+```no-highlight
+trainer(output_directory: ENV["NEVERCODE_XUNIT_RESULTS_DIR"])
+```
+
+## Publish your build artefacts
+
+You can use either `fastlane` to take care of artefact distribution or choose from a number of [Nevercode's own integartions](https://developer.nevercode.io/docs/what-kind-of-build-distribution-services-does-nevercode-support).
```

Comment:
`integrations` instead of `integartions`

Reply:
🙈done.